docs/toolbelt-catalog.yaml

```diff
@@ -0,0 +1,41 @@
+# Catalog entry for Backstage [backstage.io]
+
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+    name: ci-framework
+    title: ci-framework
+    description: |
+        CI Framework - used for CI, QE and Devs to run OSP 18+ jobs in a converged way
+    annotations:
+        github.com/project-slug: openstack-k8s-operators/ci-framework
+        feedback/type: JIRA
+        feedback/host: https://issues.redhat.com
+        jira/project-key: OSPRH
+    links:
+      - title: docs
+        url: https://ci-framework.readthedocs.io/en/latest/
+        icon: docs
+      - title: code
+        url: https://github.com/openstack-k8s-operators/ci-framework
+        icon: github
+      - title: #osp-podified-ci-support
+        url: https://app.slack.com/client/E030G10V24F/C03MD4LG22Z
+        icon: chat
+    tags:
+        - testing
+        - test-execution
+        - test-framework
+        - test-management
+        - test-reporting
+        - provisioning
+        - python
+        - openstack
+        - openshift
+        - cloud
+        - continuous-integration
+    namespace: quality-community
+spec:
+    type: tool
+    owner: group:redhat/openstack-k8s-operators-ci
+    lifecycle: production
```

Comment:
Do you have a link to what this metadata actually means. Everything looks correct to me, I would just like to know what `lifecycle: production` actually means because ci-framework is not a customer product... if that's what it means but as I say idk what it means :)

Reply:
It doesn't mean that. Catalog is purely for our QE, so they can look up the best tools for the job.

Reply:
In that case it's okay for me